src/libraries/System.Runtime.InteropServices/tests/TestAssets/NativeExports/ComInterfaceGeneratorExports.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.ObjectiveC;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Runtime.InteropServices.ComWrappers;
+
+namespace NativeExports;
+
+public static unsafe class ComInterfaceGeneratorExports
+{
+    interface IComInterface1
+    {
+        public int GetData();
+
+        public void SetData(int x);
+
+        public static Guid IID = new Guid("2c3f9903-b586-46b1-881b-adfce9af47b1");
+    }
+
+    // Call from another assembly to get a ptr to make an RCW
+    [UnmanagedCallersOnly(EntryPoint = "get_com_object")]
+    public static void* CreateComObject()
+    {
+        MyComWrapper cw = new();
+        var myObject = new MyObject();
+        nint ptr = cw.GetOrCreateComInterfaceForObject(myObject, CreateComInterfaceFlags.None);
+
+        return (void*)ptr;
+    }
+
+    class MyComWrapper : System.Runtime.InteropServices.ComWrappers
+    {
+        protected override ComInterfaceEntry* ComputeVtables(object obj, CreateComInterfaceFlags flags, out int count)
+        {
+            if (obj is IComInterface1)
+            {
+                ComInterfaceEntry* comInterfaceEntry = (ComInterfaceEntry*)RuntimeHelpers.AllocateTypeAssociatedMemory(typeof(MyObject), sizeof(ComInterfaceEntry));
+                comInterfaceEntry->IID = IComInterface1.IID;
+                void** vtable = (void**)RuntimeHelpers.AllocateTypeAssociatedMemory(typeof(ComInterfaceGeneratorExports), sizeof(void*) * 5);
+                GetIUnknownImpl(out var fpQueryInterface, out var fpAddReference, out var fpRelease);
+                vtable[0] = (void*)fpQueryInterface;
+                vtable[1] = (void*)fpAddReference;
+                vtable[2] = (void*)fpRelease;
+                vtable[3] = (delegate* unmanaged<void*, int*, int>)&MyObject.ABI.GetData;
+                vtable[4] = (delegate* unmanaged<void*, int, int>)&MyObject.ABI.SetData;
+                comInterfaceEntry->Vtable = (nint)vtable;
+                count = 5;
```

Comment:
```suggestion
                count = 1;
```

We're only defining one vtable here, so our count is 1.